### DISCUSSION TITLE
Carry over attributes when creating Issues from a task list

### DISCUSSION THREAD

**Discussion by st-pasha** [community/community#4544] [Projects and Issues]
Suppose issue N has a task list, and I'm converting that task list into individual issues.
It would be helpful if the newly created issues carried over attributes (if present) from the original issue:
- assignee 
- projects
- milestones

If you ever decide to ask for confirmation before creating a new issue, then the modal window can have checkboxes for which of the attributes to copy.

**Comment by abernix** [comment]
I think this is a great discussion but seems like a duplicate of https://github.com/github/feedback/discussions/4330.  Maybe better to 👍 up that?

**Reply by st-pasha** [reply]
Indeed, we can do both.
Right now they haven't even put the [Task lists] label on either of these discussions, which means they haven't been seen by the developers yet.

**Reply by abernix** [reply]
How about https://github.com/github/feedback/discussions/4690?

**Comment by Highvsm** [comment]
Hi i need help with something on github im a newbie this the only way to
ask help

**Comment by eli-crow** [comment]
I think especially projects and milestones. Assignees would not be wanted in my case. Maybe this could be configurable in automations?
